index.d.ts

```diff
@@ -759,6 +753,11 @@ declare namespace Eris {
   }
 
   // Invite
+  interface CreateChannelInviteOptions extends CreateInviteOptions {
+    targetApplicationID?: string;
+    targetType?: 1 | 2;
```

Comment:
```suggestion
    targetType?: Constants["InviteTargetTypes"][keyof Constants["InviteTargetTypes"]];
```

This isn't in dev yet, but it should be soon

(you can add them in early if you want to avoid errors [[index.d.ts](https://github.com/DonovanDMC/eris/blob/constants/index.d.ts#L1661-L1664)] [[Constants.js](https://github.com/DonovanDMC/eris/blob/constants/lib/Constants.js#L272-L275)])